pandas/core/generic.py

```diff
@@ -10210,6 +10210,44 @@ def _doc_parms(cls):
 nan
 """
 
+_max_examples = """\
+Examples
+--------
+``MultiIndex`` series example of monthly rainfall
+
+>>> index = [np.tile(['London', 'New York'], 3),
+...          np.repeat(['Jun', 'Jul', 'Aug'], 2)]
+>>> s = pd.Series([47, 112, 35, 117, 54, 113], index=index)
+>>> s.rename_axis(['city', 'month'], inplace=True)
```

Comment:
Slight preference for `s = s.rename_axis(...)` rather than `inplace=True`.

Actually, can you move this to the Index construction? I'd fine that clearer.

```python
>>> index = pd.MultiIndex.from_arrays(
...     [np.tile(['London', 'New York'], 3),
...      np.repeat(['Jun', 'Jul', 'Aug'], 2)],
...     names=['city', 'month'])
```

should do the trick

Reply:
Done